src/fyrox/beginning/scene_and_scene_graph.md

```diff
@@ -1,87 +1,79 @@
+- [Particle system](../scene/particle_system_node.md) - allows you to create visual effects using a huge set of small particles. It
+  can be used to create smoke, sparks, blood splatters, etc..
+- [Terrain](../scene/terrain_node.md) - allows you to create complex landscapes with minimal effort.
+- [Decal](../scene/decal_node.md) - paints on other nodes using a texture. It is used to simulate cracks in concrete walls,
   damaged parts of the road, blood splatters, bullet holes, etc.
-- [Rigid Body](../physics/rigid_body.md) - a physical entity that is responsible for dynamic of the rigid. There is a special variant
-for 2D - `RigidBody2D`.
-- [Collider](../physics/collider.md) - a physical shape for a rigid body, it is responsible for contact manifold generation, 
-without it any rigid body will not participate in simulation correctly, so every rigid body must have at least
+- [Rigid Body](../physics/rigid_body.md) - a physical entity that is responsible for the dynamic of the rigid. There is a special 
+variant for 2D - `RigidBody2D`.
+- [Collider](../physics/collider.md) - a physical shape for a rigid body. It is responsible for contact manifold generation, 
+without it, any rigid body will not participate in simulation correctly, so every rigid body must have at least
 one collider. There is a special variant for 2D - `Collider2D`.
-- [Joint](../physics/joint.md) - a physical entity that restricts motion between two rigid bodies, it has various amounts
+- [Joint](../physics/joint.md) - a physical entity that restricts motion between two rigid bodies. It has various amounts
 of degrees of freedom depending on the type of the joint. There is a special variant for 2D - `Joint2D`.
-- [Rectangle](../scene/rectangle.md) - a simple rectangle mesh that can have a texture and a color, it is a very simple version of 
+- [Rectangle](../scene/rectangle.md) - a simple rectangle mesh that can have a texture and a color. It is a very simple version of 
 a Mesh node, yet it uses very optimized renderer, that allows you to render dozens of rectangles simultaneously.
-This node is intended to be used for **2D games** only.
-- [Sound](../sound/sound.md) - a sound source, it is universal for 2D and 3D. Spatial blend factor allows you to select
+This node is intended for use in **2D games** only.
+- [Sound](../sound/sound.md) - a sound source universal for 2D and 3D. Spatial blend factor allows you to select
 a proportion between 2D and 3D.
-- Listener - an audio receiver that captures the sound in a particular point in your scene and sends it to an audio
-context for processing and output to an audio playback device.
-- Animation Player - a container for multiple animations, it can play animations made in the 
+- Listener - an audio receiver that captures the sound at a particular point in your scene and sends it to an audio
+context for processing and outputing to an audio playback device.
```

Comment:
typo `outputing`

Reply:
Fixed them, I must have forgotten run this file through a spell checker